### PR TITLE
feat: make turso db tokens not expire by default

### LINKED ISSUE #337: Make `turso db tokens create` tokens not expire by default
People will run `turso db tokens create`, which expires by default. Let's make a non-expiring token by default.

### PR DESCRIPTION
**Pull request by StefanoSaffran**
This PR aims to update de expiration flag options and make it not expire by default.

image

closes #337

### CODE REVIEW

File: internal/cmd/db_generatetoken.go [hunk 1]
```diff
@@ -12,7 +13,7 @@ var readOnly bool
 func init() {
 	dbTokensCmd.AddCommand(dbGenerateTokenCmd)
 
-	usage := "Token expiration. Possible values are 'default' or 'none'."
+	usage := fmt.Sprintf("Token expiration. Possible values are %s (default) or expiration time in days (e.g. %s).", internal.Emph("never"), internal.Emph("7d"))
```
Comment (haaawk) [inline internal/cmd/db_generatetoken.go:16]: I would stick to none instead of never. Even though never is a more clear.

Comment (haaawk) [reply internal/cmd/db_generatetoken.go:16]: I would allow more time units not just days.

Comment (StefanoSaffran) [reply internal/cmd/db_generatetoken.go]: As said in the API's PR, kept it with days for now and changed back `never` to `none`.

File: internal/cmd/expiration_flag.go [hunk 2]
```diff
@@ -2,30 +2,35 @@ package cmd
...
-	if slices.Contains(expFlagValues, v) {
+
+	if strings.HasSuffix(v, "d") {
+		checkIfNumber := strings.TrimSuffix(v, "d")
+		_, err := strconv.Atoi(checkIfNumber)
+		if err != nil {
+			return fmt.Errorf("must be %s or expiration time in days (e.g. %s)", internal.Emph("none"), internal.Emph("7d"))
+		}
 		*e = expirationFlag(v)
 		return nil
```
[the remark this answers is not in the document]
Comment (StefanoSaffran) [reply internal/cmd/expiration_flag.go]: done!

File: internal/cmd/expiration_flag.go [hunk 3]
```diff
@@ -2,30 +2,34 @@ package cmd
...
-		*e = expirationFlag(v)
-		return nil
+
+	if strings.HasSuffix(v, "d") {
+		checkIfNumber := strings.TrimSuffix(v, "d")
+		if _, err := strconv.Atoi(checkIfNumber); err == nil {
+			*e = expirationFlag(v)
+			return nil
+		}
+		return fmt.Errorf("must be %s or expiration time in days (e.g. %s)", internal.Emph("none"), internal.Emph("7d"))
```
Comment (haaawk) [inline internal/cmd/expiration_flag.go:30]: You don't need this return.

**Review by haaawk — APPROVED** [review]

**Pull request merged**